var/spack/repos/builtin/packages/xsdk/package.py

```diff
@@ -43,7 +43,9 @@ class Xsdk(Package):
     version('0.3.0', 'a52dc710c744afa0b71429b8ec9425bc', preferred=True)
     version('xsdk-0.2.0', 'a52dc710c744afa0b71429b8ec9425bc')
 
-    variant('debug', default=False, description='Compile in debug mode')
```

Comment:
I think this was a placeholder for a xsdk feature - being able to install debug or optimized variants of all xsdk pacakges. Sure - it does not work currently - but don't know what xsdk group would want to do with this.

cc: @jwillenbring , @keitat

Reply:
> I think this was a placeholder for a xsdk feature 

I just saw that it's not being used, so I thought one can remove it.

I can keep it as-is and drop this change, it's fine with me either way.

Reply:
Fine with me either way. Just wanted relavent folks to know about this change.

We can keep this change for now - and  add thisfeature  back when we know how to implement it.